packages/vue3-vite/src/generators/docs/files/README.md.template

```diff
@@ -1,6 +1,6 @@
-# <% projectTitle %> Documentation
+# <%= projectTitle %> Documentation
```

Comment:
I accidentally found that some equal signs are missing in the template files of docs.

Reply:
Good catch!